libraries/Microsoft.Bot.Builder.Ai.LUIS/Client/LuisService.cs

```diff
@@ -204,14 +204,18 @@ public interface ILuisService
     [Serializable]
     public sealed class LuisService : ILuisService
     {
+        private static HttpClient g_httpClient = new HttpClient();
```

Comment:
The .NET design rule for statics is to name them with `TitleCase` and I think that's what we've got across the majority of the code base at this point. Typically I would name this something like `private static HttpClient DefaultHttpClient`. I would also consider making it a `Lazy<>` since, if people are bringing their own, there's no reason to pay the startup time and allocation/memory overhead.